zubhub_frontend/zubhub/src/views/project_details/ProjectDetails.jsx

```diff
@@ -491,12 +507,57 @@ function ProjectDetails(props) {
             })
           }
           aria-labelledby={t('projectDetails.ariaLabels.imageDialog')}
-        >
-          <img
-            className={classes.enlargedImageStyle}
-            src={enlarged_image_url}
-            alt={`${project.title}`}
-          />
+        > 
+         {/* Cancel Button on the Preview Images */}
+          <IconButton
+            className={classes.cancelEnlargedImageBtn}
+            onClick={() =>
+              setState({
+                ...state,
+                open_enlarged_image_dialog: !open_enlarged_image_dialog,
+              })
+            }
+            aria-label="close"
+          >
+            <CloseIcon style={{ color: 'white' }} />
+          </IconButton>
+
+            {/* Slider Image Container */}
+          <Box className={classes.enlargedImageContainer}>
+          {project.images.length <= 1 ? (
+            <img
+              className={classes.enlargedImageStyle}
+              src={enlarged_image_url}
+              alt={`${project.title}`}
+            />
+          ) : (
+              <Slider
+                initialSlide={enlarged_image_id} //Start the slider from the image clicked in projectDetails
+                adaptiveHeight
+                infinite
+                speed={500}
+                slidesToShow={1}
+                slidesToScroll={1}
+                className= {classes.enlargedImageStyle}
+                nextArrow = {<EnlargedImgArrow/>}
+                prevArrow = {<EnlargedImgArrow/>}
+
+              >
+                {project.images.map(image => (
+                  <div>
+                    <img
+                      style={{
```

Comment:
Use classes instead of inline styles

Reply:
And images need to have alt text, If the image doesn't have title or description properties add empty text for alt